spec/lang/step/terminators.md

```diff
@@ -174,6 +174,25 @@ impl<M: Memory> Machine<M> {
         })
     }
 
+    /// A helper function to evaluate the return expression and prepare it for in-place passing.
+    fn eval_return_place(&mut self, ret_expr: PlaceExpr) -> NdResult<(Place<M>, Type)> {
+        // First evaluate the return place. (Left-to-right!)
```

Comment:
This comment makes little sense here, it needs to stay in `eval_terminator` where the order of evaluation is defined.